### ISSUE TITLE
Question: Action Notify, problem with Sns message

### ISSUE THREAD

**Issue by SebastianZZ** [cloud-custodian/cloud-custodian#5012]
Hi !

I have created policy with notify action and sns topic in AWS. I want to customize my message because now im getting something looks like a group of random letters. Any idea how to do this ? I can't see any information in docs. 

` policies:
  - name: my-first-policy
    resource: aws.ec2
    filters:
      - "tag:project" : present
      - "State.Name" : running 
    actions:
      - type: notify
        subject: "CUSTODIAN - [{{ account }} {{ region }}]"
        to:
          - cloudcustodian_notify_test
        transport:
          type: sns
          topic: cloudcustodian_notify_test
          region: eu-central-1
          attributes:
            Name: 'AAAAAAAAA'
            Message: 'asdasdasdasdas'
            body: 'asdasdasdasdasda' `

**Comment by SebastianZZ** [comment]
Yeah I know but I don't want to use SES, i am in Frankfurt and its not available here.

> The to list specifies the intended recipient for the email. You can specify either an email address, an 
SNS topic, a Datadog Metric, or a special value. 

and as u can see, I have added 
`        to:
          - cloudcustodian_notify_test`

but for some reason it is now working for me.

**Comment by kapilt** [comment]
mailer is required and can templated messages to a variety of transports ie. sns/smtp/ses/etc

the sns transport for the notify action is a data channel. you have to configure mailer to listen on the other side (sns->sqs->mailer) to get it delivered/rendered for humans.
